src/scripts/run.sh

```diff
@@ -5,73 +5,76 @@ airflow initdb  # initdb is not destructive, so can be re-run at startup
 python scripts/mkvars.py
 
 # creating an admin and regular users (nessacary when using RABC=True in the airflow.cnf)
-airflow create_user -r Admin -u admin -e admin@example.com -f admin -l admin -p ${AIRFLOW_USER_ADMIN_PASSWD}
-airflow create_user -r User -u dataservices -e dataservices@example.com -f dataservices -l dataservices -p ${AIRFLOW_USER_DATASERVICES_PASSWD}
-airflow create_user -r User -u team_ruimte -e team_ruimte@example.com -f team_ruimte -l team_ruimte -p ${AIRFLOW_USER_TEAM_RUIMTE_PASSWD}
+airflow users create -r Admin -u admin -e admin@example.com -f admin -l admin -p ${AIRFLOW_USER_ADMIN_PASSWD:-admin}
```

Comment:
De admin user wordt reeds met dit script aangemaakt, getuige deze regel. In de README.md staat echter dat deze nog aangemaakt moet worden met een expliciet commando. Dat kan dus uit de README gehaald worden.